boardswarm-cli/README.md

```diff
@@ -34,3 +45,12 @@ The UI has the follow keyboard shortcuts:
 * ^a k: Scroll up
 * ^a j: Scroll down
 * ^a 0: Reset scrolling state
+
+## Boardswarm device control
+
+Basic board control can be performed without opening the tui, for example
+a board can be powered on and off:
```

Comment:
Mind that's only the case if the mode are configure like on/off  ofcourse; those names are just conventions